astropy/coordinates/representation.py

```diff
@@ -1589,28 +1874,48 @@ def _get_base_vectors(cls, base):
         return base.unit_vectors(), base.scale_factors()
 
     def to_cartesian(self, base):
-        """Convert the differential to 3D rectangular cartesian coordinates.
+        """
+        Convert the differential to 3D rectangular cartesian coordinates.
 
         Parameters
         ----------
         base : instance of ``self.base_representation``
              The points for which the differentials are to be converted: each of
              the components is multiplied by its unit vectors and scale factors.
+
+        Returns
+        -------
+        This object as a `CartesianDifferential`
         """
-        base_e, base_sf = self._get_base_vectors(base)
-        return functools.reduce(
-            operator.add, (getattr(self, d_c) * base_sf[c] * base_e[c]
-                           for d_c, c in zip(self.components, base.components)))
+        return self._to_cartesian_helper(base)
 
     @classmethod
     def from_cartesian(cls, other, base):
-        """Interpret 3D rectangular cartesian coordinates as differentials.
+        """
+        Convert the differential to 3D rectangular cartesian coordinates.
```

Comment:
Yea, I have to remember. I personally don't like that style so my go-to is the line break, but will try to be conscious of the pep.